src/common/components/cards/DeckSummary.tsx

```diff
@@ -1,31 +1,33 @@
 import * as React from 'react';
-import { arrayOf, bool, func, object } from 'prop-types';
 import Badge from 'material-ui/Badge';
 import Paper from '@material-ui/core/Paper';
 import FontIcon from 'material-ui/FontIcon';
 import { filter, sortBy } from 'lodash';
 
-import { TYPE_ROBOT, TYPE_EVENT, TYPE_STRUCTURE } from '../../constants.ts';
-import { groupCards } from '../../util/cards.ts';
-import { BUILTIN_FORMATS } from '../../util/formats.ts';
+import * as w from '../../types';
+import { TYPE_ROBOT, TYPE_EVENT, TYPE_STRUCTURE } from '../../constants';
```

Comment:
So a `js` file requiring `ts` modules needs the extension but `tsx` files dont'?

Reply:
Yeah, typescript files don't need it when referring to other typescript files.